diff --git a/pkg/k8s/test/addressresolver_cases.go b/pkg/k8s/test/addressresolver_cases.go
index ca8ed516..233d6ebd 100644
--- a/pkg/k8s/test/addressresolver_cases.go
+++ b/pkg/k8s/test/addressresolver_cases.go
@@ -121,7 +121,7 @@ func knService(namespace string) ResolveAddressTestCase {
 		ObjectMeta: metav1.ObjectMeta{
 			Name: m.name, Namespace: namespace, Labels: labels,
 		},
-		Spec: ksvcSpec("gcr.io/knative-samples/helloworld-go"),
+		Spec: ksvcSpec("quay.io/openshift-knative/helloworld-go:multiarch"),
 		Status: servingv1.ServiceStatus{
 			RouteStatusFields: servingv1.RouteStatusFields{
 				URL: serviceURL,
